clustered/client/src/main/java/org/ehcache/clustered/client/internal/store/ClusteredStore.java

```diff
@@ -175,32 +174,12 @@ protected String getStatisticsTag() {
   }
 
```

Comment:
You've altered the behavior of when compaction is going to be triggered: it is only happening on atomic ops and on client requests now. I don't think this is bad, but I wonder if that shouldn't be documented in a comment or something.

Reply:
I don't think I did... or at least I didn't intend to... I'll reinvestigate and try to confirm that I succeeded.